spec/setup.sh

```diff
@@ -0,0 +1,57 @@
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#! /bin/bash
+
+if [ $# -ne 1 ]; then
+    echo $0: usage: setup.sh STEP_NAME
+    exit 1
+fi
+
+STEP_NAME=$1
+
+TEST_DIR=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )/../$STEP_NAME
+
+# use example database config
```

Comment:
It's not wrong - we are using `database.example.yml`. I'll clarify the comment though.